lib/Service/ObjectService.php

```diff
@@ -112,12 +212,14 @@ public function getObject(Register $register, string $uuid): ObjectEntity
 
 	* @return ObjectEntity The resulting object.
 	*/
-   public function deleteObject(Register $register, string $uuid)
+   public function deleteObject(Register $register, Schema $schema, string $uuid): bool
    {
 	// Lets see if we need to save to an internal source
-	if ($register->getSource() === 'internal') {
-	   $object = $this->objectEntityMapper->findByUuid($uuid);
+	if ($register->getSource() === 'internal' || $register->getSource() === '') {
```

Comment:
in_array would be shorter
in_array($register->getSource(), ['internal', ''])

Or maybe use empty($register->getSource()) instead of $register->getSource() === ''

Reply:
But not necessarily more readable. This also is temporary until we can choose sources correctly